.abs/main.yaml

```diff
@@ -0,0 +1,6 @@
+chart-dir: helm/aws-node-termination-handler
+replace-chart-version-with-git: true
+replace-app-version-with-git: false
```

Comment:
I think I always set this to true, any reason to disable it?

Reply:
just left it in case we want to reference the upstream docker image, this would break if we fork the image for whatever reason though...